scripts/checkTranslation.js

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable no-console */
+const yaml = require("js-yaml");
+const fs = require("node:fs");
+const path = require("node:path");
+
+function loadYaml() {
+  const locales = path.resolve(__dirname, "../src/locales");
+  const dirs = fs.readdirSync(locales, { withFileTypes: true });
+  const all = [];
+
+  for (const dir of dirs) {
+    const out = {};
+    out.lang = dir.name;
+    out.data = {};
+
+    if (!dir.isDirectory()) {
+      continue; //  probably a system file like .DS_STORE
+    }
+
+    const langDir = path.resolve(locales, dir.name);
+    const dirAndFiles = fs.readdirSync(langDir, { withFileTypes: true });
+
+    for (const entry of dirAndFiles) {
+      if (entry.isDirectory()) {
+        // handle nested folders?
```

Comment:
to get this to work with nested files, extract out the for of loop above and do it recursively.